### DISCUSSION TITLE
Alpescraft 2023 🏔️

### DISCUSSION THREAD

**Discussion by vgallet** [lyonjs/shortvid.io#147] [Conference / Meetup Communication]
Let's see what we can do to help "Alpescraft" conference ;-)

**Comment by Slashgear** [comment]
@vgallet 
Let's see what we can do :) 

What are your usage of social video generator ? 

- Sponsor announce videos ? 
- Speaker / Talk announce ?
- Other ?

We will need to have assets and colors used in your events to setup a _Design_ 

Either you can give a static JSON for data or an API.

Take a look at the discussion we had with Mixit orga in #100 to see what we can do

@vgallet 👋🏻

**Comment by vgallet** [comment]
Trop bien!

Yes! We can use it for sponsor announce videos and speaker / talk announce. I can not provide a static JSON data nor API so I think I will have to use manually the generator to create the videos and gif.

Our first need is to announce speakers & talks.

Here's the assets:
 - the logo https://www.alpescraft.fr/img/logo/color/03-AlpesCraft_Couleurs-M.png.
 - the background image we used to use https://www.alpescraft.fr/img/cover.webp
 - the font colors we used, on white backgound #212529 and on colored background #fff
 - the color we used to decorate the website #007bff 

Don't hesitate to tell me if the images are not in the right format or too small.


Thank you very much! I own you some :beers:

**Reply by mickaelalvs** [reply]
Hey ! 👋🏼 Est ce que tu as une date pour laquelle vous en aurez besoin ?

**Reply by vgallet** [reply]
^^ ASAP

**Reply by vgallet** [reply]
we will start to announce some speakers so as soon as you can get it done. Thank you very much

**Comment by Boyadjie** [comment]
Hi @vgallet !
here is a first version of the video to announce the conference what do you think ?
I'm working on another one for the talk & speakers with a similar style.

https://user-images.githubusercontent.com/72607059/222378974-e0e720f7-a2ef-4495-963d-a3e58da34285.mp4

**Reply by vgallet** [reply]
wonderful!

**Reply by Boyadjie** [reply]
I'm happy you like it !

Do you know if there will be talks with 3 speakers ?

**Reply by vgallet** [reply]
I can not answer this yet, it's possible

**Comment by Boyadjie** [comment]
Here is the version with the speakers : 

https://user-images.githubusercontent.com/72607059/222470370-d4450ad7-008a-4c81-93e4-f249055d9e5e.mp4

**Reply by Slashgear** [reply]
Space between speaker pictures is a bit huge (to me) and they are maybe to close from the top.
Otherwise this looks really nice 💪🏻 
Did we test with huge speaker names and a huge conf name ?

**Reply by Boyadjie** [reply]
yes that why there is a lot of spaces around the title, i've lower a bit the pictures and reduce the space between them it look like that at the end : 
<img width="868" alt="Screenshot 2023-03-02 at 16 57 43" src="https://user-images.githubusercontent.com/72607059/222481814-a7ad8092-a059-49a6-b663-ad020d2bcbd5.png">

**Reply by vgallet** [reply]
la classe à dallas! :clap:

**Reply by vgallet** [reply]
is it possible to hide some parts if no text is prodived? I mean, if I don't provide a schedule time, is the clock display ?

**Reply by Slashgear** [reply]
Normally yes, we could do that.

**Comment by vgallet** [comment]
Thank you very much!
